elastio-stream-kafka/consumer.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+import argparse
+import logging
+import subprocess
+
+from kafka import KafkaConsumer
+from common import id_generator, store_last_message, store_first_message
+
+
+logging.basicConfig(level=logging.CRITICAL)
+
+backup_parser = argparse.ArgumentParser(
+    prog="Elastio stream kafka consumer"
+)
+backup_parser.add_argument("--topic_name", type=str, nargs="?", help="Enter Kafka topic name to backup.")
+backup_parser.add_argument("--brokers", type=str, nargs="+", help="Enter one or more Kafka brokers separated by spaces.")
+args = backup_parser.parse_args()
+
+# parse brokers and topic name from the script arguments
+bootstrap_servers = args.brokers
+topic_name = args.topic_name
+_id = id_generator()
+
+consumer = KafkaConsumer(
+    topic_name,
+    group_id=f'{_id}-group',
+    bootstrap_servers=bootstrap_servers,
+    auto_offset_reset='earliest', # latest/earliest
+    enable_auto_commit=True,
+    auto_commit_interval_ms=1000, # 1s 
+    consumer_timeout_ms=10000, # 10s
+    api_version=(2,0,2)
+    )
+
+res = subprocess.run(
+    ['elastio', 'rp', 'list', '--output-format', 'json'],
+    stdout=subprocess.PIPE, stderr=subprocess.PIPE
```

Comment:
Again, this can hang if you're not reading stderr. If you don't want stderr, you should not pass `PIPE`, and should instead either pass `subprocess.DEVNULL` to fully ignore it, or pass nothing to forward it to your own stderr.